testsuite/features/step_definitions/common_steps.rb

```diff
@@ -22,17 +22,79 @@
   $server.run("umount #{iso_path}; mount #{iso_path}")
 end
 
-Then(/^I can see all system information for "([^"]*)"$/) do |host|
+Then(/^the hostname for "([^"]*)" should be correct$/) do |host|
   node = get_target(host)
   step %(I should see a "#{node.hostname}" text)
+end
+
+Then(/^the kernel for "([^"]*)" should be correct$/) do |host|
+  node = get_target(host)
   kernel_version, _code = node.run('uname -r')
-  puts 'I should see kernel version: ' + kernel_version
   step %(I should see a "#{kernel_version.strip}" text)
+end
+
+Then(/^the OS version for "([^"]*)" should be correct$/) do |host|
+  node = get_target(host)
   os_version, os_family = get_os_version(node)
   # skip this test for centos and ubuntu systems
   step %(I should see a "#{os_version.gsub!('-SP', ' SP')}" text) if os_family.include? 'sles'
 end
 
+Then(/^the IPv4 address for "([^"]*)" should be correct$/) do |host|
+  node = get_target(host)
+  ipv4 = client_public_ip(host)
```

Comment:
Will this be ever run for terminals? If so see Oscar's PR for private_ip use here

Reply:
Ah right, maybe is better that you rebase master, then you will see available a `node.public_ip` attribute

Reply:
Done. I now use the attribute Oscar mentioned.